packages/apollo-link-batch-http/README.md

```diff
@@ -0,0 +1,59 @@
+# Batch HTTP Link

```

Comment:
I think we want all of these to be in the docs site - maybe we can symlink so that it's both a README and a docs page?